transforms/append-px-to-style-properties.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+'use strict';
+
+let itemsToNotSuffixWithPx = [
+  'animationIterationCount',
+  'boxFlex',
+  'boxFlexGroup',
+  'boxOrdinalGroup',
+  'columnCount',
+  'fillOpacity',
+  'flex',
+  'flexGrow',
+  'flexPositive',
+  'flexShrink',
+  'flexNegative',
+  'flexOrder',
+  'fontWeight',
+  'lineClamp',
+  'lineHeight',
+  'opacity',
+  'order',
+  'orphans',
+  'stopOpacity',
+  'strokeDashoffset',
+  'strokeOpacity',
+  'strokeWidth',
+  'tabSize',
+  'widows',
+  'zIndex',
+  'zoom',
+];
+
+const shouldSuffixIfIntergerValue = attr => itemsToNotSuffixWithPx.indexOf(attr) === -1;
+
+const getStyle = attrs => attrs.find(attr => attr.name && attr.name.name === 'style');
+
+function transformProperties(props, root) {
+  props.forEach(p => {
+    if (p.type === 'SpreadProperty') {
+      transformStyleObject(p.argument, root);
+    } else {
+      const key = p.key.type === 'Identifier' ? p.key.name : p.key.value;
+      let val = p.value.value;
+      if (val === undefined) {
+        if (p.value.type === 'UnaryExpression') {
+          val = parseInt(`${p.value.operator}${p.value.argument.value}`, 10); 
+          if (shouldSuffixIfIntergerValue(key) && typeof val === 'number') {
+            p.value.operator = '';
+            p.value.argument.value = `${val}px`;
```

Comment:
I'm surprised that printing this works, but I guess a `UnaryExpression` without an operator just prints the value?